pkgs/development/tools/sigrok-firmware-fx2lafw/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, stdenv
+, fetchurl
+, sdcc
+}: stdenv.mkDerivation (finalAttrs: {
```

Comment:
```suggestion
}:

stdenv.mkDerivation rec {
```

Reply:
Done. Is there a recommendation/guideline whether to prefer `finalAttrs` or `rec`? It appeared to me that `finalAttrs` was meant to overcome `rec` and was preferred.

Reply:
``finalAttrs`` has no usecase for a fixed output derivation.